your_daily_dose_malware/main.py

```diff
@@ -1,44 +1,49 @@
 import os
-
-import click
-from commands.scraper import scraper
+import argparse
+from rich.console import Console
+from your_daily_dose_malware.commands.utils.retrieve_malware import (all_most_recent,
+                                                                     hundred_most_recent)
 from dotenv import load_dotenv
-from commands.utils.retrieve_malware import (
-                            all_most_recent, hundred_most_recent)
 
 
 load_dotenv()
 
 
-@click.command(
-    help="""
-    Download either hundred recent malware's uploaded within the last 60 min or
-    all the recent malware's uploaded within the last 48 hours sha256 hash
-    by using api from 'https://mb-api.abuse.ch/api/v1/'
-
-    -s256 or --by-sha256: get the most recent sha256 hashes from
-    'https://bazaar.abuse.ch/export/txt/sha256/recent' save them in
-    sha256_names.txt then for each file  download all malware's in zip file
-
-    -hr or --hundred-recent: get the most recent hundred recent malware's
-    uploaded within the last 60 min
-    """
-)
-@click.option("-s256", "--by-sha256", is_flag=True)
-@click.option("-hr", "--hundred-recent", is_flag=True)
-def run_scrapper(by_sha256, hundred_recent):
-    headers = {"API-KEY": os.getenv("API_KEY")}
+def scrap_malware_bazaar(hundred_recent):
+    headers = {"API-KEY": os.getenv("MwBz_API_KEY")}
```

Comment:
Nit: Should `MwBz_API_KEY` be part of constants file?

Reply:
it's a good questions cause now we are using me own api keys that i generated from mwb website. does each user should create their own before to use ? we can my own for now.

Reply:
Each user should generate their own API keys, and load it on their own environment variables to be loaded.